<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilenameFormatWizardPage</class>
 <widget class="QWizardPage" name="FilenameFormatWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Format info</string>
  </property>
  <property name="subTitle">
   <string>Set format name and information about files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Format name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Log filename format</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leFilenameRegex">
     <property name="text">
      <string notr="true">log-{module}-{time}</string>
     </property>
     <property name="placeholderText">
      <string>Wildcard</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>File extension</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leExtension">
     <property name="text">
      <string notr="true">.log</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Encoding</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbEncoding">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
